读取第 0 张头像，名称为 20200922041640213_4bf39a2e.png
ConnectionError(MaxRetryError("HTTPConnectionPool(host='127.0.0.1', port=12242): Max retries exceeded with url: /imr-ai-service/atomic_functions/faces_detect (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000015C40CEB6C8>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))"))
特征向量已提取。
耗时 0:00:02.259651
读取第 1 张头像，名称为 20200927074522915_7f1dce40.png
ConnectionError(MaxRetryError("HTTPConnectionPool(host='127.0.0.1', port=12242): Max retries exceeded with url: /imr-ai-service/atomic_functions/faces_detect (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000015C40926EC8>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))"))
特征向量已提取。
耗时 0:00:02.265793
读取第 2 张头像，名称为 20200927075356220_3e9b3067.png
ConnectionError(MaxRetryError("HTTPConnectionPool(host='127.0.0.1', port=12242): Max retries exceeded with url: /imr-ai-service/atomic_functions/faces_detect (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000015C4144F3C8>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))"))
特征向量已提取。
耗时 0:00:02.268624
读取第 3 张头像，名称为 20200927094924872_ea5e4e91.png
ConnectionError(MaxRetryError("HTTPConnectionPool(host='127.0.0.1', port=12242): Max retries exceeded with url: /imr-ai-service/atomic_functions/faces_detect (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000015C4146DD08>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))"))
特征向量已提取。
耗时 0:00:02.279266
读取第 4 张头像，名称为 20201009104315342_fc3a206f.png
ConnectionError(MaxRetryError("HTTPConnectionPool(host='127.0.0.1', port=12242): Max retries exceeded with url: /imr-ai-service/atomic_functions/faces_detect (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000015C4148E988>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))"))
特征向量已提取。
耗时 0:00:02.265427
读取第 5 张头像，名称为 20201009104334100_de4d58c0.png
ConnectionError(MaxRetryError("HTTPConnectionPool(host='127.0.0.1', port=12242): Max retries exceeded with url: /imr-ai-service/atomic_functions/faces_detect (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000015C414AF448>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))"))
特征向量已提取。
耗时 0:00:02.270230
读取第 6 张头像，名称为 20201009104401370_a3a8afcb.png
ConnectionError(MaxRetryError("HTTPConnectionPool(host='127.0.0.1', port=12242): Max retries exceeded with url: /imr-ai-service/atomic_functions/faces_detect (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000015C414C7D88>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))"))
特征向量已提取。
耗时 0:00:02.259539
读取第 7 张头像，名称为 20201013031059652_a8b7d778.png
ConnectionError(MaxRetryError("HTTPConnectionPool(host='127.0.0.1', port=12242): Max retries exceeded with url: /imr-ai-service/atomic_functions/faces_detect (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000015C414ED9C8>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))"))
特征向量已提取。
耗时 0:00:02.307818
读取第 8 张头像，名称为 20201014025243950_065f0251.png
ConnectionError(MaxRetryError("HTTPConnectionPool(host='127.0.0.1', port=12242): Max retries exceeded with url: /imr-ai-service/atomic_functions/faces_detect (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000015C4150F408>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))"))
特征向量已提取。
耗时 0:00:02.255023
读取第 9 张头像，名称为 20201019032258986_70a8108e.png
ConnectionError(MaxRetryError("HTTPConnectionPool(host='127.0.0.1', port=12242): Max retries exceeded with url: /imr-ai-service/atomic_functions/faces_detect (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000015C414BF848>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))"))
特征向量已提取。
耗时 0:00:02.247938
读取第 10 张头像，名称为 20201019032347045_39089b7b.png
ConnectionError(MaxRetryError("HTTPConnectionPool(host='127.0.0.1', port=12242): Max retries exceeded with url: /imr-ai-service/atomic_functions/faces_detect (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000015C4146DA88>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))"))
特征向量已提取。
耗时 0:00:02.243122
读取第 11 张头像，名称为 20201021044105469_1025f99a.png
ConnectionError(MaxRetryError("HTTPConnectionPool(host='127.0.0.1', port=12242): Max retries exceeded with url: /imr-ai-service/atomic_functions/faces_detect (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000015C4144F788>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))"))
特征向量已提取。
耗时 0:00:02.281028
读取第 12 张头像，名称为 20201021105910178_eaca442c.png
ConnectionError(MaxRetryError("HTTPConnectionPool(host='127.0.0.1', port=12242): Max retries exceeded with url: /imr-ai-service/atomic_functions/faces_detect (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000015C626AB808>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))"))
特征向量已提取。
耗时 0:00:02.268759
读取第 13 张头像，名称为 20201021110012718_c7a45f04.png
ConnectionError(MaxRetryError("HTTPConnectionPool(host='127.0.0.1', port=12242): Max retries exceeded with url: /imr-ai-service/atomic_functions/faces_detect (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000015C414DEC88>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))"))
特征向量已提取。
耗时 0:00:02.263506
读取第 14 张头像，名称为 20201021110144861_08493f93.png
ConnectionError(MaxRetryError("HTTPConnectionPool(host='127.0.0.1', port=12242): Max retries exceeded with url: /imr-ai-service/atomic_functions/faces_detect (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000015C414EB188>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))"))
特征向量已提取。
耗时 0:00:02.250438
读取第 15 张头像，名称为 20201021110230709_31668f31.png
ConnectionError(MaxRetryError("HTTPConnectionPool(host='127.0.0.1', port=12242): Max retries exceeded with url: /imr-ai-service/atomic_functions/faces_detect (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000015C4148E0C8>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))"))
特征向量已提取。
耗时 0:00:02.264042
读取第 16 张头像，名称为 20201022090924506_0972d227.png
ConnectionError(MaxRetryError("HTTPConnectionPool(host='127.0.0.1', port=12242): Max retries exceeded with url: /imr-ai-service/atomic_functions/faces_detect (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000015C41513448>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))"))
特征向量已提取。
耗时 0:00:02.249052
读取第 17 张头像，名称为 20201023041746532_ee9075a7.png
ConnectionError(MaxRetryError("HTTPConnectionPool(host='127.0.0.1', port=12242): Max retries exceeded with url: /imr-ai-service/atomic_functions/faces_detect (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000015C415329C8>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))"))
特征向量已提取。
耗时 0:00:02.295039
读取第 18 张头像，名称为 20201026111442350_84380902.png
ConnectionError(MaxRetryError("HTTPConnectionPool(host='127.0.0.1', port=12242): Max retries exceeded with url: /imr-ai-service/atomic_functions/faces_detect (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000015C40CE0BC8>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))"))
特征向量已提取。
耗时 0:00:02.278807
读取第 19 张头像，名称为 20201101011059185_7347a6d3.png
ConnectionError(MaxRetryError("HTTPConnectionPool(host='127.0.0.1', port=12242): Max retries exceeded with url: /imr-ai-service/atomic_functions/faces_detect (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000015C40CE2C08>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))"))
特征向量已提取。
耗时 0:00:02.249472
读取第 20 张头像，名称为 20201101080236160_40cf9903.png
ConnectionError(MaxRetryError("HTTPConnectionPool(host='127.0.0.1', port=12242): Max retries exceeded with url: /imr-ai-service/atomic_functions/faces_detect (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000015C40B6AE08>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))"))
特征向量已提取。
耗时 0:00:02.249184